client/src/Footer.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+import {
+  Segment,
+  Container,
+  Grid,
+  List,
+  Divider,
+  Header,
+  Icon,
+  Image,
+} from 'semantic-ui-react';
+
+const Footer = () => (
+  <div>
+    <Segment style={{ margin: '5em 0em', padding: '5em 0em' }} vertical style={{ background: '#353E48' }}>
+      <Container>
+        <Grid columns={4} stackable>
+          <Grid.Row>
+            <Grid.Column>
+              <a href="#" style={{ textDecoration: 'underline', color: '#fff' }}>Start a new group</a>
+            </Grid.Column>
+          </Grid.Row>
+          <Divider />
+          <Grid.Row>
+            <Grid.Column>
+              <Header inverted as="h4" content="Your Account" />
+              <List link inverted>
+                <List.Item as="a">Settings</List.Item>
+                <List.Item as="a">Log Out</List.Item>
+                <List.Item as="a">Help</List.Item>
+              </List>
+            </Grid.Column>
+            <Grid.Column>
+              <Header inverted as="h4" content="Discover" />
+              <List link inverted>
+                <List.Item as="a">Groups</List.Item>
+                <List.Item as="a">Calendar</List.Item>
+                <List.Item as="a">Topics</List.Item>
+                <List.Item as="a">Cities</List.Item>
+              </List>
+            </Grid.Column>
+            <Grid.Column>
+              <Header inverted as="h4" content="Meetup" />
+              <List link inverted>
+                <List.Item as="a">About</List.Item>
+                <List.Item as="a">Meetup Pro</List.Item>
+                <List.Item as="a">Careers</List.Item>
+                <List.Item as="a">Apps</List.Item>
+                <List.Item as="a">API</List.Item>
+                <List.Item as="a">Accessibility</List.Item>
+              </List>
+            </Grid.Column>
+            <Grid.Column>
+              <Header inverted as="h4" content="Follow us" />
+              <List horizontal link inverted>
```

Comment:
const socialsIcons = ["facebook official", "twitter" etc]

socialsIcons.forEach---etc

Is there an issue with working with semantic ui lib? Or is this a first pass?

Reply:
Could these have been other components to iterate over?

Reply:
> Could these have been other components to iterate over?
Definitely. 

> const socialsIcons = ["facebook official", "twitter" etc]
> 
> socialsIcons.forEach---etc
> 
> Is there an issue with working with semantic ui lib? Or is this a first pass?
First pass, no issues with semantic ui, using a loop in navbar.